.github/workflows/windows.yml

```diff
@@ -53,22 +53,25 @@ jobs:
         unzip -o "mecab-msvc-x64.zip" -d c:/mecab
     - name: Install dependencies
       run: |
-        python -m pip install --upgrade setuptools wheel pip setuptools-scm
+        python -m pip install --upgrade setuptools wheel pip setuptools-scm delvewheel
         pip install -r requirements.txt
     - name: Build wheel
       run: |
-        python setup.py bdist_wheel
+        python -m build --wheel
```

Comment:
`python setup.py ...` command is deprecated.
https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html

Reply:
I have had this pointed out to me and tried to transition before, but for a long time after people started saying setup.py usage was deprecated, it turned out that there was no usable alternative for libraries with C extensions. I would be delighted if this just works now.